activestorage/test/controllers/representations/redirect_controller_test.rb

```diff
@@ -60,8 +60,7 @@ class ActiveStorage::Representations::RedirectControllerWithVariantsWithStrictLo
     follow_redirect!
     assert_match(/^inline/, response.headers["Content-Disposition"])
 
-    @blob.reload # became free of strict_loading?
-    image = read_image(@blob.variant(resize_to_limit: [100, 100]))
+    image = read_image_from_response
```

Comment:
note to self: `ActiveStorage::Representations::RedirectController`

`@representation.record.blob` doesn't match the variant blob?